install/linux/usr/share/odemis/sim/enzel-sim.odm.yaml

```diff
@@ -285,6 +286,17 @@ ENZEL-Sim: {
   # used.
 }
 
+# Special component to provide control of the 5DOF in a global reference frame
+"Global 5DOF Aligner": {
+  class: actuator.MultiplexActuator,
+  role: global5DOF,
```

Comment:
@pieleric do you think that is needed? Probably since I need this component for the alignment tab I will discover there I need to add it to MainGUIData.

Reply:
Indeed, you'll need it in MainGUIData. You can do it in a separate PR though. However, so far for the roles the rule has been all lowercases, with words separated by a "-". So I'd suggest something like "stage-global".